### PR TITLE
feat(slack): add kubernetes-streaming slack channel

### PR DESCRIPTION
**Pull request by kris-nova**
Signed-off-by: Kris Nova <kris@nivenly.com>

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by markyjackson-taulia** [comment]
Can you provide more context for this channel?

File: communication/slack-config/channels.yaml [hunk 1]
```diff
@@ -174,6 +174,7 @@ channels:
   - name: kubernetes-operators
   - name: kubernetes-ruby
   - name: kubernetes-security
+  - name: kubernetes-streaming
```
Comment (idvoretskyi) [inline communication/slack-config/channels.yaml]: @kris-nova can you provide more details on this channel?

**Comment by kris-nova** [comment]
Sure - basically there have been a few folks who wanted to start talking about bringing various Kubernetes streams from across the internet together while most of the world is at home. This channel would be the community for planning some of the collaboration here.

Basically - imagine a `hack the planet` style loose-association of live streams that try to schedule near each other - without overlapping. 

I don't want to prescribe too much - because I think this would ultimately be planned and governed in the open. But there was enough interest I figured we should an open space to begin talking about it - and if it doesn't pan out we can always archive it.

**Comment by markyjackson-taulia** [comment]
Could we not use the announcement channel?

**Comment by justaugustus** [comment]
`#announcements` is read-only for Slack admins

**Comment by kris-nova** [comment]
yeah - ideally this would behave like a small project or a loose network - but we want a place where anyone can get involved - right now there has been a DM and that doesn't seem to open/friendly enough (and also leads to backchanneling, hence why we need a place in the open to point folks to)

**Comment by markyjackson-taulia** [comment]
My gut tells me there would need to be some process and/or governance (I could be wrong). As a slack admin I feel this might need more planning to ensure we are not overlooking

**Comment by castrojo** [comment]
/lgtm

+1 from me on this, we have a good overlap of people in k8s who also do other streams, it would be nice to have a place for everyone to coordinate and share best practices. 

This could also be a good place for the youtube-admins to hang out and discuss things like equipment, services, tools, etc instead of cluttering up the main contrib-ex channel.

**Comment by jdumars** [comment]
/approve
/hold
I am ok trying this, and if problems arise adjust accordingly. My only ask is to have a link pinned to the channel that explains exactly why it is there, so it stays on point.  Someone else from the admin team can lift the hold as they see fit.

**Comment by kris-nova** [comment]
To start can we reference [this](https://github.com/kubernetes/community/pull/4633#issuecomment-600295982) to start - and taking our first meeting to figure out a place to document this in further detail to update the link then?

**Comment by kris-nova** [comment]
Meeps - can we remove the hold please?

**Comment by markyjackson-taulia** [comment]
sorry about that
/unhold

**Pull request merged**